### ISSUE TITLE
[Slider] When screen reader is running, the arrows does not work as expected

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#2969]
**Describe the bug**
When a screen reader is running (NVDA in this case) the arrow keys do not update the value. Keydown/keyup events are not triggered.

**To Reproduce**
1. Run NVDA screen reader
1. Use arrows to update the value of the Slider
https://stackblitz.com/edit/slider-kendoui-example-vdfvxr?file=app/app.component.ts

**Workaround**
Wrap the slider in a span with role _application_:
https://stackblitz.com/edit/slider-kendoui-example-sa6rup?file=app/app.component.ts

**Comment by preslavapetrova1** [comment]
Screenreaders provide keyboard navigation themselves, which overrides the keyboard navigation of the components. This is not an issue on our side. A workaround can be found in the following knowledge base [article](https://www.telerik.com/kendo-angular-ui/components/knowledge-base/general-keyboard-navigation-screenreaders/).
